R/dbSendQuery.R

```diff
@@ -49,7 +49,10 @@ NULL
         post.response=post.response,
         statement=statement,
         session.timezone=conn@session.timezone,
-        cursor=cursor
+        cursor=cursor,
+        update.conn.parameter=function(property, value) {
+          set_connection_parameter(conn, property, value)
```

Comment:
The RCpp route feels rather hacky. My suggestion would be to use a reference class (e.g. named PrestoSession) for storing the parameters in the connection - similar to what we are doing with the PrestoCursor. Then we can carry the instance of that class around in the PrestoResult to update as necessary.
We could have a method in PrestoConnection that returns the required list by reading the values from the reference class instance.